extensions/argparseext.py

```diff
@@ -43,9 +43,13 @@ def parse_arguments():
     initial task description. must be quoted.
     if not specified, get initial_task from environment.
     ''', default=os.getenv("INITIAL_TASK", os.getenv("FIRST_TASK", "")))
-    parser.add_argument('-4', '--gpt-4', dest='openai_api_model', action='store_const', const="gpt-4", help='''
-    use GPT-4 instead of the default GPT-3 model.
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('-4', '--gpt-4', dest='openai_api_model', action='store_const', const="gpt-4", help='''
```

Comment:
Not sure I love these arguments. Why not a single `--llm` or `-l` that takes the model? e..g., llama, gpt-3.5-turbo, etc

Reply:
Mostly because this makes it very explicit to people how to choose what are the most likely top two models folks will want. Shortcuts to OPEN_AI_MODEL= with specific values (I do agree we should rename that variable)